process_report/process_report.py

```diff
@@ -62,16 +70,45 @@ def is_old_pi(old_pi_dict, pi, invoice_month):
     return False
 
 
+def get_invoice_bucket():
+    try:
+        b2_resource = boto3.resource(
+            service_name="s3",
+            endpoint_url=os.environ.get(
+                "B2_ENDPOINT", "https://s3.us-east-005.backblazeb2.com"
```

Comment:
B2 is an implementation detail, because the interface type this communicates to is s3. Please rename your changes mentioning the environment variables with `B2_*` to `S3_*` and the variables from `b2_*` to `s3_*`.